gateway/utils/handlers/config_crud.go

```diff
@@ -560,6 +559,50 @@ func HandleInspectCollectionSchema(adminMan *admin.Manager, modules *modules.Mod
 	}
 }
 
+// HandleInspectUntrackedCollectionSchema gets the schema for particular collection & removed the collection from the database collection schema in config
+func HandleInspectUntrackedCollectionSchema(adminMan *admin.Manager, modules *modules.Modules, syncman *syncman.Manager) http.HandlerFunc {
```

Comment:
This should be named as `HandleUntrackCollectionSchema` as per me. Your naming conventions don't make sense with the action actually performed.